hbase-server/src/main/java/org/apache/hadoop/hbase/master/procedure/ServerCrashProcedure.java

```diff
@@ -484,6 +484,9 @@ private void assignRegions(MasterProcedureEnv env, List<RegionInfo> regions) thr
         }
         if (env.getMasterServices().getTableStateManager().isTableState(regionNode.getTable(),
           TableState.State.DISABLING, TableState.State.DISABLED)) {
+          env.getAssignmentManager().regionClosedAbnormally(regionNode);
```

Comment:
Do we still need to set this if the TableState is DISABLED?

Reply:
Looks like no need, let me fix it.

Reply:
Please add some comments to describe the problem here? We need to change the state here otherwise the TRSP scheduled by DTP will try to close the region from a dead server and will never succeed.

Reply:
done